src/ruleLoader.ts

```diff
@@ -52,8 +52,12 @@ export function loadRules(ruleConfiguration: {[name: string]: any},
     }
 
     if (notFoundRules.length > 0) {
-        const errorMessage = `Could not find the following rules specified in the configuration:\n${notFoundRules.join("\n")}`;
-        throw new Error(errorMessage);
+        const ERROR_MESSAGE = `
```

Comment:
since this is a template string, newlines and whitespace will be taken into account, so I think this message might end up with too much whitespace as is